rules/tines_rules/tines_story_items_destruction.py

```diff
@@ -0,0 +1,8 @@
+def rule(event):
+    return event.get('operation_name') == 'StoryItemsDestruction'
```

Comment:
please use `deep_get()` with a `default=` kwarg in place of `event.get()`

Reply:
The Tines.Audit log type has a global filter, this should be the first thing that happens inside the rule function https://github.com/panther-labs/panther-analysis/blob/master/rules/tines_rules/tines_custom_ca.py#L11-L12